Create first draft of suspect commit documentation (#19529)
DataDog/documentation: content/en/logs/error_tracking/suspect_commits.md

---
title: Suspect Commits
kind: documentation
disable_toc: false
---
## Overview

> For some issues, Error Tracking can identify a suspect commit, helping you determine the root cause and resolve the issue. This feature is automatically enabled when its [setup requirements](#setup) are met.

Reviewer: ```suggestion
Error Tracking can identify a suspect commits, helping you pinpoint the root cause of your errors and resolve issue faster. This feature is automatically enabled on issues when the [setup requirements](#setup) are met.
```

{{< img src="logs/error_tracking/suspect_commit.png" alt="A suspect commit as it is displayed in the Datadog UI" style="width:100%" >}}

> If a suspect commit is associated with an issue, it is displayed on the issue panel, as shown in the highlighted area of the image below.

Reviewer: ```suggestion
Once a suspect commit has been identified, it is displayed on the issue panel as shown in the highlighted area of the image below.
```
